### DISCUSSION TITLE
Help with using AWS SSL Certificates with OvenMediaEngine edge EC2 instance

### DISCUSSION THREAD

**Discussion by morpheus04** [OvenMediaLabs/OvenMediaEngine#1416] [Help]
Hello,

I recently configured both origin and edge EC2 instances using OvenMediaEngine on AWS and faced an issue with using TLS on both the load balancer (LB) and the edge instance. I configured the TLS listeners and attached the AWS ACM SSL certificates onto the LB. However, I wasn't able to play the stream. The TLS port 3334 is enabled on both firewall and server.xml, so no issues with ports or configs. I also added permissions via IAM role to the edge instance but had no joy with the WebRTC streams.

From the logs, the LB traffic is acknowledged by the edge server but with the error below;
```
https_server.cpp:100  | Could not handle connection event: there is no certificate
```

Please is anyone able to provide some help to this issue?

much appreciated

**Comment by getroot** [comment]
If the certificate is installed on the LB, you should try connecting to the Non-TLS port, which is OME's 3333 port. Port 3334 is the TLS port by default and requires the certificate to be installed in Server.xml.
For more detailed analysis, please upload the entire log file and Server.xml.

**Reply by morpheus04** [reply]
thanks. however ws on port 3333 works (so that wasn't the problem). the only issue is with wss on port 3334. If the certs need to be installed on the instance as indicated (unfortunately AWS doesn't allow certs registered to a domain to be exported expect they are private CA certs).
